### DISCUSSION TITLE
Item Spell Points

### DISCUSSION THREAD

**Discussion by Kharmans** [misthero/dnd5e-spellpoints#73] [Q&A]
I don't know how difficult is to do what I am about to ask, so, sorry if what I ask is impossible.

Why not configure the module to subtract Spell Point from an item named **X** from the sheet of the actor? Like, with the Resources, you could change the name of the resource to whatever you wanted. Why not just do that with this new module? You could just make that configuration to Items on the sheet.

![image](https://github.com/misthero/dnd5e-spellpoints/assets/112586547/9627148b-a423-4d86-8e3b-fb625f531984)

I just change the name on the module and have an item named exactly that on the sheet. Then, the module subtract the points of that item. It's better to support languages other than English and you can create the Item with any name you wanted.

**Comment by misthero** [comment]
check latest version please (only if you are on dndv3+)

**Comment by Kharmans** [comment]
I check and it only function with the Spell Points item of the module. Or it function with any item?

**Comment by Kharmans** [comment]
Okay, I just see what you are saying, but I think I am making something wrong. I already have the Class Feature named Pontos de Mana, but it doesn't work.

![image](https://github.com/misthero/dnd5e-spellpoints/assets/112586547/43ad0658-a842-4798-92f6-fd5a7316c9a8)
![image](https://github.com/misthero/dnd5e-spellpoints/assets/112586547/fd8638d5-cacd-49d6-8b0f-ecde0e9fc0f3)
![image](https://github.com/misthero/dnd5e-spellpoints/assets/112586547/5d1516eb-9fab-4d51-b112-87ea05503724)
[fvtt-Item-pontos-de-mana-Rz4kGwWe3swGnLJQ.json](https://github.com/misthero/dnd5e-spellpoints/files/14579251/fvtt-Item-pontos-de-mana-Rz4kGwWe3swGnLJQ.json)

**Comment by Kharmans** [comment]
Even if I change the name of the Spell Points item to Pontos de Mana, it doesn't recognize it.

It only function with the name **Spell Points**.

**Comment by misthero** [comment]
you need to change (if you want to use your own item) the "source custom label". Not the name. The item name can be anything.

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/b80a8601-c12c-44fe-b819-3293ed7e686b)


![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/008388c3-2f88-40f6-b9d5-7874719b5b6c)


The compendium of the module have an item already configured.

**Comment by misthero** [comment]
maybe I have to explain that more clearly, where it say "class feature" it should say "feature" item...

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/48a70fcc-9b14-4e9b-a7c2-2cade11f5fcb)

**Comment by misthero** [comment]
oh, one more thing. when you create or drop the item on the character sheet, it get saved and only that item will work.

to change item you need to remove it from the sheet and add a new one.

this is necessary to find the right item for that character.

I need to make this behaviour transparent to the user to avoid confusion.

I will work on that.

**Comment by Kharmans** [comment]
When I change the custom label to Pontos de Mana, it worked!
Thanks and sorry for the dumb question hehe
